README.md

```diff
@@ -45,6 +45,9 @@ OpenDevin agents collaborate with human developers to write code, fix bugs, and
 ![App screenshot](./docs/static/img/screenshot.png)
 
 ## ⚡ Getting Started
+
+The easiest way to run OpenDevin is to [![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://github.com/codespaces/new/OpenDevin/OpenDevin) but it is not the recommended usage path yet. For the best experience and optimal setup, please follow the docker method.
+
```

Comment:
This is very confusing to be part of the main README.

Reply:
Agree, it looks strange to put a not recommend way at the beginning of main readme.